plugins/woocommerce-admin/client/marketplace/components/search/search.scss

```diff
@@ -20,7 +20,7 @@
 	}
 
 	@media (width <= $breakpoint-medium) {
-		margin: $content-spacing-small;
+		margin: $grid-unit-20 $grid-unit-20 $grid-unit-10 $grid-unit-20;
```

Comment:
Appreciate that you're switching over to the standard variables instead of our custom ones. That was my hope over time!